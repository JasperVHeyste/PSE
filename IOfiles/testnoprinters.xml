<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<JOB>
		<jobNumber>1</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
		<type>bw</type>
	</JOB>
	<JOB>
		<jobNumber>2</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>color</type>
	</JOB>
	<JOB>
		<jobNumber>3</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>scan</type>
	</JOB>
</SYSTEM>