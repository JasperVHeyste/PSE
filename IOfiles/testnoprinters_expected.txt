No printers present in Printingsystem

